includes/compatibility/class-wc-date-compatibility.php

```diff
@@ -44,7 +44,7 @@ public function __toString()
      * @return int
      * @since  3.0.0
      */
-    public function getTimestamp()
+    public function getTimestamp(): int
     {
         return method_exists('DateTime', 'getTimestamp') ? parent::getTimestamp() : $this->format('U');
```

Comment:
```suggestion
        return method_exists('DateTime', 'getTimestamp') ? parent::getTimestamp() : time();
```

Reply:
@joerivanveen That change would return the current time and doesn't take in account any values set on the object.

Reply:
@michielgerritsen very sharp.